Makefile

```diff
@@ -11,5 +11,5 @@ watch:
 
 env:
 	@echo 'Creating local environment....'
-	virtualenv --python=python3.6 --no-download env
-	$(PIP) install -r requirements.txt
\ No newline at end of file
+	virtualenv --python=python3.8 --no-download env
```

Comment:
Can we use python3 here? On Macs it is difficult to have more than one version of python 3 installed. This will allows individuals to have a version of python installed but not necessarily the latest. Unless 3.8 is absolutetly required

Reply:
Actually just found out about a tool called pyenv to manage python versions. I'm going to give it a try tomorrow.

Reply:
I'll leave this as `3.8` for the time being.